### PR TITLE
Implements award_bounty, award_swag, and suggest_award

### PR DESCRIPTION
**Pull request by esjee**
Fixes https://github.com/oreoshake/hackerone-client/issues/21

Usage:

```
> report = HackerOne::Client::Api.new('$HANDLE').report($REPORT_ID)
> report.award_bounty(message: "Here's your bounty!", amount: 500, bonus_amount: 50)
> report.suggest_bounty(message: "I suggest $500 with a small bonus. Report is well-written.", amount: 500, bonus_amount: 50)
> report.award_swag(message: "Here's your T-Shirt")
```

(For bounty and suggest bounty, either `amount` or `bonus_amount` is required, not both.)

### CODE REVIEW

**Comment by Willianvdv** [comment]
Nice! 💸

**Comment by esjee** [comment]
Please note you have to make sure your API user has the right permission to create bounties. Otherwise the API will return with a 404 error.

2017-08-22-171323_727x348_scrot

**Comment by oreoshake** [comment]
Thanks @esjee! I probably won't get to this until next week.

File: spec/hackerone/client/report_spec.rb [hunk 1]
```diff
@@ -98,6 +98,55 @@
...
+    it 'creates a bounty' do
+      result = VCR.use_cassette(:award_swag) do
+        report.award_swag(
+          message: 'Enjoy this cool swag!',
+        )
+      end
+
+      expect(result).to be_a HackerOne::Client::Swag
+      expect(result.sent).to eq false
+    end
```
Comment (mvgijssel) [inline spec/hackerone/client/report_spec.rb]: It seems that this API request is not tested?

Comment (esjee) [reply spec/hackerone/client/report_spec.rb]: It is; this just happens to be an attribute of the swag object. "Sent" indicates whether or not the swag was sent; the user can toggle this in the HackerOne interface. I added a test for the `created_at` attribute.

**Pull request merged**